### ISSUE TITLE
Unhandled JS Exception: error is not defined

### ISSUE THREAD

**Issue by ythecombinator** [opentok/opentok-react-native#44]
I've been struggling with this error:

<img width="500" alt="captura de tela 2018-04-23 as 17 21 07" src="https://user-images.githubusercontent.com/2644563/39151146-c0f0c8da-471a-11e8-9579-82dc400cda5d.png">

Which results in this being print on the simulator:

<img width="200" alt="captura de tela 2018-04-23 as 15 35 07" src="https://user-images.githubusercontent.com/2644563/39150544-0e10a830-4719-11e8-8be2-044964ec9819.png">

I spent a little time checking the source of this lib and it seems that `handleError` defined [here](https://github.com/opentok/opentok-react-native/blob/master/src/OTError.js) is being called [here](https://github.com/opentok/opentok-react-native/blob/master/src/OTPublisher.js#L65) without any parameter so error is `undefined`.

Turns out that I can't even manage to find any error itself given that everything I pass down to OpenTok components is okay.

This is the source:

```typescript
// ...
    const {openTok} = environment.settings;
    const {session, token} = this.props.details;
// ...
        {(this.props.status === Status.connected || this.props.status === Status.connecting) && (
          <View style={styles.callContainer}>
            <OTSession apiKey={openTok.apiKey} sessionId={session} token={token}>
              <OTPublisher
                style={{width: 100, height: 100}}
                properties={{
                  publishAudio: this.state.publishAudio,
                  cameraPosition: this.state.cameraPosition,
                }}
                eventHandlers={{
                  streamCreated: this.onRoomDidConnect,
                }}
              />
              <OTSubscriber style={{width: 100, height: 100}} />
            </OTSession>
//...
```

And logging stuff shows that my props are valid:

<img width="500" alt="captura de tela 2018-04-23 as 17 40 11" src="https://user-images.githubusercontent.com/2644563/39152049-67dfcff4-471d-11e8-98a4-68050545d7bd.png">

Any ideas of what's going on?

**Comment by msach22** [comment]
@ythecombinator Could you please let me know which version of the library you're using? There was an issue with the `OTPublisher` component where the `error` parameter was being used, but it should've been `pulbishError`. If you use v0.4.1, the error should be handled correctly.

**Comment by ythecombinator** [comment]
Just checked my `package.json` file and it is `v0.4.0`–had installed about one week ago. Updated to `v0.5.0` and the error seems to be gone. I'll keep testing here and, if necessary, I reopen this.

Btw, sorry for opening another one; it's just that searching the issues I couldn't find any related–now I found #25–perhaps a more expressive title would help.

Thanks for the great support and quick feedback 😀–it always helps a lot!
